src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -29,18 +29,19 @@
     @RequestMapping("/")
     public String index(Model model) {
         Item item = new Item(0, "item");
-        item.setDescription("Des");
-        item.setLocation("locale");
-        item.setAssignedTo(1);
-        item.setStatusId(0);
+        //item.setDescription("Des");
+        //item.setLocation("locale");
+        //item.setAssignedTo(1);
+        //item.setStatusId(0);
         model.addAttribute(item);
         return "index";
     }
+    //
 
 
     @RequestMapping("/saveItem")
-    public String saveItem(Item item) {
-        //itemService.save(item);
+    public String saveItem(Item item) throws Exception {
```

Comment:
Is an exception actually thrown here? 
I don't see one.
If not, then adding "throws Exception" to the method signature is misinforming the user.
If so, be specific about which exception is thrown.